modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/to/Chapter.kt

```diff
@@ -5,14 +5,12 @@ import kotlin.math.roundToInt
 
 data class Chapter(
     val title: String,
-    val url: HttpUrl?,
     var startTime: Int,
     var endTime: Int,
-    val imagePath: String?,
-    val mimeType: String?,
-    var played: Boolean = false,
-    var index: Int = 0,
-    var progress: Float = 0.0f
```

Comment:
It didn't feel right storing the `progress` and `played` state in the chapter object as they can change regularly based on the playback position.